HISTORY.rst

```diff
@@ -12,21 +12,27 @@ History
 
 0.6.2 (2023-03-22)
 ------------------
+
```

Comment:
This file is not up to date, please pull master branch

Reply:
Ok, it is updated now.